docs/roles/OBORoles.md

```diff
@@ -0,0 +1,23 @@
+# OBO Foundry Roles
+
+OBO Foundry Roles are designed to manage a single aspect of the day-to-day activities of the OBO Foundry. 
+This document presents the SOPs for role recruitment and retirement, and the expectations and responsibilities common to all roles.
+
+## General SOPs for role recruitment and retirement
+
+- The duration of an appointment to a role is 1 year, with the possibility of extension.
+- Role assignments will be reviewed during the first OBO Operations Committee call that falls in December each year.
+- Any OBO Operations Committee member may nominate a candidate for a given role. Alternatively, candidates can be self-nominated. New volunteers are particularly encouraged to participate.
+- Candidates for each role should read this document and all documents pertaining to the role of interest, acknowledge that they understand the involved duties, and agree to fulfill them to the best of their ability.
+- Each candidate must submit an application for a role to the OBO Operations Committee. The application should outline the motivation for taking on the role as well as 
+  describe the skillsets that will make them suitable for that role.
+- Applications will be reviewed by the OBO Operations Committee. Decisions for appointing a member of the community to a role will be based on vote. A positive outcome occurs (that is, the appointment is approved) if the candidate receives at least 70% in favor.
```

Comment:
I assume only current OBO Ops Committee members can vote, even though random members of the public will be allowed to sit in on Ops committee meetings.
By 70%, do you mean 70% of those who vote, or 70% of the whole ops committee? Like, if the ops committee has 20 people, and only 10 come to the meeting and vote, and 7 of them vote yes, is that sufficient? or do you need 14 yeses?

Reply:
We typically do those present, though if we went that route then we should probably require a minimum number of people for the vote to be valid.

Reply:
```suggestion
- Applications will be reviewed by the OBO Operations Committee. Decisions for appointing a member of the community to a role will be based on vote _during an OBO Operations Call_. A positive outcome occurs (that is, the appointment is approved) if the candidate receives at least 70% votes in favor by OBO Operations Committee members present during the OBO Operations Call.
```

Reply:
What about this?